bin/docker/alpine/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN bin/build
 FROM alpine:3.6
 
 # Install packages
-RUN apk add --update --no-cache iptables ca-certificates openvpn bash sudo openresolv
+RUN apk add --update --no-cache iptables ipset ca-certificates openvpn bash sudo openresolv
```

Comment:
You are adding new dependencies thus it is necessary to do `[ci full]` build on this PR.